data/en/mts_0001.json

```diff
@@ -0,0 +1,611 @@
+    "form_type": "0001",
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0001",
+    "title": "Monthly Turnover Survey – Retail Sales",
+    "theme": "default",
+    "legal_basis": "Notice is given under section 1 of the Statistics of Trade Act 1947.",
+    "navigation": {
+        "visible": false
+    },
+    "metadata": [{
+            "name": "user_id",
+            "validator": "string"
+        },
+        {
+            "name": "period_id",
+            "validator": "string"
+        },
+        {
+            "name": "trad_as_or_ru_name",
+            "validator": "string"
+        },
+        {
+            "name": "ru_name",
+            "validator": "string"
+        },
+        {
+            "name": "trad_as",
+            "validator": "string"
+        },
+        {
+            "name": "ref_p_start_date",
+            "validator": "date"
+        },
+        {
+            "name": "period_str",
+            "validator": "string"
+        },
+        {
+            "name": "ref_p_end_date",
+            "validator": "date"
+        }
+    ],
+    "sections": [{
+            "id": "introduction-section",
+            "title": "Introduction",
+            "groups": [{
+                "blocks": [{
+                    "type": "Introduction",
+                    "id": "introduction",
+                    "primary_content": [{
+                        "id": "get-started",
+                        "content": [{
+                            "list": ["On average it takes ten minutes to complete this survey once you’ve collected the information.", "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.", "You can provide informed estimates if actual figures aren’t available.", "We will treat your data securely and confidentially."]
+                        }]
+                    }],
+                    "preview_content": {
+                        "id": "preview",
+                        "title": "Information you need",
+                        "content": [{
+                            "title": "",
+                            "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                        }],
+                        "questions": [{
+                            "question": "Retail Sales including VAT",
+                            "content": [{
+                                "description": "Include:",
+                                "list": ["VAT", "online sales"]
+                            }, {
+                                "description": "Exclude:",
+                                "list": ["sales from catering facilities used by customers (for example, in-store cafes)", "revenue from mobile phone network commission and top-up", "NHS receipts", "lottery sales and commission from lottery sales", "sales of car accessories and motor vehicles"]
```

Comment:
My OCD is annoyed that the order of these is wrong. I won't block it on this but it hurts.